RNTester/js/PickerExample.js

```diff
@@ -116,7 +116,7 @@ class PickerExample extends React.Component<{}, $FlowFixMeState> {
     this.setState({mode: newMode});
   };
 
-  onValueChange = (key: string, value: string) => {
+  onValueChange = (key: string, value: string | number) => {
```

Comment:
I get this error....
```js
Cannot create Picker element because string [1] is incompatible with number [2] in the
first argument of property onValueChange.

     RNTester/js/PickerExample.js
      37│     return (
      38│       <RNTesterPage title="<Picker>">
      39│         <RNTesterBlock title="Basic Picker">
      40│           <Picker
      41│             style={styles.picker}
      42│             selectedValue={this.state.selected1}
                      // this line causes error
      43│             onValueChange={this.onValueChange.bind(this, 'selected1')}>
      44│             <Item label="hello" value="key0" />
      45│             <Item label="world" value="key1" />
      46│           </Picker>
      47│         </RNTesterBlock>
      48│         <RNTesterBlock title="Disabled picker">
      49│           <Picker
        :
 [1] 119│   onValueChange = (key: string, value: string) => {

     Libraries/Components/Picker/Picker.js
 [2]  73│   onValueChange?: ?(itemValue: string | number, itemIndex: number) => mixed,
```